UnitTests/UnitTests.HighPerformance.Shared/Enumerables/Test_ReadOnlyRefEnumerable{T}.cs

```diff
@@ -47,6 +47,53 @@ public void Test_ReadOnlyRefEnumerable_DangerousCreate_BelowZero(int length, int
         {
             _ = ReadOnlyRefEnumerable<int>.DangerousCreate(in Unsafe.NullRef<int>(), length, step);
         }
+
+        [TestCategory("ReadOnlyRefEnumerable")]
+        [TestMethod]
+        [DataRow(1, 1, new[] { 1 })]
+        [DataRow(4, 1, new[] { 1, 2, 3, 4 })]
+        [DataRow(4, 4, new[] { 1, 5, 9, 13 })]
+        [DataRow(4, 5, new[] { 1, 6, 11, 16 })]
```

Comment:
Nit: you can remove the `length` input parameter, as it's redundant and makes the test more error prone. That value is always the same as the length of the input array, so you might as well just use that in the test methods 🙂
Same for this and the other test methods below as well.